.repo-metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "google-auth-library-python",
+  "name_pretty": "Google Auth Python Library",
+  "client_documentation": "https://googleapis.dev/python/google-auth-library-python/latest",
+  "issue_tracker": "https://github.com/googleapis/google-auth-library-python/issues",
+  "release_level": "beta",
```

Comment:
Is it really only beta? I would hope we'd have GAed our auth lib by now.

Reply:
Looks like it is GA (according to the setup.py). Thank you for catching that! https://github.com/googleapis/google-auth-library-python/blob/master/setup.py